duckduckgo/duckduckgo-autofill
# Remove initial dismissal in favor of single permanent dismissal

**Reviewer:** @shakyShane @GioSensation 
**Asana:** https://app.asana.com/0/0/1204038361216227/f

## Description
Remove initial dismissal in favour of single permanent dismissal

![incontext-signup-dismiss](https://user-images.githubusercontent.com/635903/221874425-86323f59-1515-4aad-b136-df7e8bce0db6.gif)

## Steps to test
1. Load branch autofill into extension (branch [`abrown/bugfix-default-pixel-params`](https://github.com/duckduckgo/duckduckgo-privacy-extension/compare/abrown/bugfix-default-pixel-params)) and enable feature toggle
2. Go to https://duckduckgo.com/newsletter
3. Confirm in-context signup shows with "Don't Ask Again"
4. Click "Don't Ask Again" and confirm it's removed from the page
5. Refresh the page and confirm it doesn't show again
